### PR TITLE
[SPARK-29093][ML][PySpark][Follow-Up] Remove duplicate setter

### PR DESCRIPTION
**Pull request by huaxingao**
### What changes were proposed in this pull request?
remove duplicate setter in ```BucketedRandomProjectionLSH``` 

### Why are the changes needed?
Remove the duplicate ```setInputCol/setOutputCol``` in ```BucketedRandomProjectionLSH``` because these two setter are already in super class ```LSH```

### Does this PR introduce any user-facing change?
No

### How was this patch tested?
Manually checked.

### CODE REVIEW

File: python/pyspark/ml/feature.py [hunk 1]
```diff
@@ -415,18 +415,6 @@ def setBucketLength(self, value):
...
-        Sets the value of :py:attr:`inputCol`.
-        """
-        return self._set(inputCol=value)
-
-    def setOutputCol(self, value):
-        """
-        Sets the value of :py:attr:`outputCol`.
-        """
-        return self._set(outputCol=value)
-
```
Comment (huaxingao) [inline python/pyspark/ml/feature.py]: These two setters are already in super class, so no need to have them here.

**Comment by huaxingao** [comment]
cc @zhengruifeng

File: python/pyspark/ml/classification.py [hunk 2]
```diff
@@ -286,56 +286,48 @@ def setParams(self, featuresCol="features", labelCol="label", predictionCol="pre
     def _create_model(self, java_model):
         return LinearSVCModel(java_model)
 
-    @since("2.2.0")
```
Comment (dongjoon-hyun) [inline python/pyspark/ml/classification.py]: This is the correct version, isn't it? If this is correct, it seems that we don't need to remove.
> In docs of all the previous versions, setMaxIter in LinearSVC doesn't have version info.

**Review by dongjoon-hyun — CHANGES_REQUESTED** [review]
Showing the correct information is a desirable behavior change.
For the removal of redundant functions, that can be a follow-up.
BTW, is the duplication caused at SPARK-29093?

File: python/pyspark/ml/classification.py (hunk 2 above)
Comment (huaxingao) [reply python/pyspark/ml/classification.py]: Yes. The versions are correct.

**Comment by huaxingao** [comment]
@dongjoon-hyun
Thanks Dongjoon. I will remove the duplication only. The duplication was introduced in SPARK-29093.

File: python/pyspark/ml/regression.py [hunk 3]
```diff
@@ -2067,7 +2067,7 @@ def setTol(self, value):
         """
         return self._set(tol=value)
 
-    @since("2.2.0")
```
Comment (huaxingao) [inline python/pyspark/ml/regression.py]: This is a typo. should be 2.0.0.

Comment (dongjoon-hyun) [reply python/pyspark/ml/regression.py]: Right.
- https://spark.apache.org/docs/2.0.0/api/python/pyspark.ml.html#pyspark.ml.regression.GeneralizedLinearRegression.setFitIntercept

**Review by dongjoon-hyun — APPROVED** [review]

**Review by viirya — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Merged to master. Thank you, @huaxingao and @viirya .

**Comment by huaxingao** [comment]
Thank you very much! @dongjoon-hyun @viirya

**Pull request closed without merging**